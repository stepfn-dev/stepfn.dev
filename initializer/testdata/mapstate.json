{
  "StartAt": "ExampleMapState",
  "States": {
    "ExampleMapState": {
      "Type": "Map",
      "Iterator": {
        "StartAt": "CallLambda",
        "States": {
          "CallLambda": {
            "Type": "Task",
            "Resource": "arn:aws:states:::lambda:invoke",
            "Parameters": {
              "Payload.$": "$",
              "FunctionName": "unix"
            },
            "End": true
          }
        }
      },
      "End": true
    }
  }
}
