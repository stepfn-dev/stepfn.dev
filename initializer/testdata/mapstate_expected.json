{
  "StartAt": "ExampleMapState",
  "States": {
    "ExampleMapState": {
      "Type": "Map",
      "Iterator": {
        "StartAt": "CallLambda",
        "States": {
          "CallLambda": {
            "Type": "Task",
            "Resource": "arn:aws:states:::lambda:invoke",
            "Parameters": {
              "Payload.$": "$",
              "FunctionName": "funcarn",
              "ClientContext": "eyJjdXN0b20iOnsiSWQiOiJkeW5hbW9pZCIsIkhhbmRsZXIiOiJ1bml4In19"
            },
            "End": true
          }
        }
      },
      "End": true
    }
  }
}
